docs/zh-CN/spark/installation.md

```diff
@@ -14,7 +14,10 @@ https://github.com/apache/incubator-seatunnel/releases
 
 ### 准备Spark
 
-`Seatunnel` 依赖`Spark`。安装`seatunnel`前,需要准备好`Spark`。请先 [下载 Spark](https://spark.apache.org/downloads.html) , 选择 `Spark 版本 >= 2.x.x`。 在下载和解压后, 不需要任何配置就可以指定 `deploy-mode = local` 提交任务。 如果需要使用其他模式，如 `Standalone cluster`，`Yarn cluster` ，`Mesos cluster`, 请参考官方文档。
+`Seatunnel` 支持`Spark`引擎。如果希望在`Spark`中使用`SeaTunnel`，需要提前准备好`Spark`环境。[下载 Spark](https://spark.apache.org/downloads.html) , 选择 `Spark 版本 >= 2.x.x 
+`，目前还不支持spark3.x。 在下载和解压后, 
+不需要任何配置就可以指定 
+`deploy-mode = local` 提交任务。 如果需要使用其他模式，如 `Standalone cluster`，`Yarn cluster` ，`Mesos cluster`, 请参考官方文档。
```

Comment:
Also, please pay attention to your document format, no line breaks are required here.